schema.sql

```diff
@@ -11,3 +11,37 @@ CREATE TABLE animals (
     PRIMARY KEY (id)
 
 );
+
+/* add two new tables.. owners and species. */
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY NOT NULL,
+    full_name varchar(100) NOT NULL,
+    age INT
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name varchar(100) NOT NULL
+);
+
+/*remove species coloum in animal table. */
+ALTER TABLE animals
+DROP COLUMN species
+
+/*adding two columns to be used as foreign reference keys*/
+
+ALTER TABLE animals
+    ADD species_id INT,
+    ADD CONSTRAINT species_id FOREIGN KEY(species_id) REFERENCES species(id);
+
+ALTER TABLE animals
+    ADD owners_id INT,
+    ADD CONSTRAINT owners_id FOREIGN KEY(owners_id) REFERENCES owners(id);
+
+
+
+
+
+
+
+
```

Comment:
- [ ] [Optional] You can just remove this unused extra-space, so that your codes will be clean